test/over_react_test/common_component_util_test.dart

```diff
@@ -27,5 +28,13 @@ main() {
         const $PropKeys(PropsThatShouldBeForwarded),
       ]);
     });
+
+    group('should pass when the correct required props are specified', () {
+      commonComponentTests(() => TestCommonRequired()..bar = true,
+          shouldTestRequiredProps: true,
```

Comment:
Well, I didn't want to re-run the other tests that are tested in the first `group`.

I'm fine either way so if it is more than a #nit and a "do it", I'll do so. :)